### ISSUE TITLE
Error on new Ledger Version with Linux

### ISSUE THREAD

**Issue by ChiefGyk** [LedgerHQ/ledger-live-desktop#3568]
#### Ledger Live Version and Operating System

Ledger Live v2.21.3
- tested on Ledger Live **version_here**
- Platform and version: Pop!_OS Linux 18.04 (fork of Ubuntu)

#### Expected behavior

No error
#### Actual behavior

Getting an error
#### Steps to reproduce the behavior

I had to reset my wallet as the upgrade broke it, and I have constantly been getting these errors every step of the way but I can restore my wallet and such fine it seems
![image](https://user-images.githubusercontent.com/19499446/107827347-9a662d00-6d54-11eb-8c69-7796801ce8c4.png)

**Comment by ChiefGyk** [comment]
Yeah it's been persisting non stop throughout the day regardless of what I do. But it hasn't impeded functionality

**Comment by gre** [comment]
Can you tell us precisely what action you are doing? You are adding accounts? of which currency? thanks

**Comment by ChiefGyk** [comment]
> Can you tell us precisely what action you are doing? You are adding accounts? of which currency? thanks

that's the thing nothing specific just ANY interaction with my ledger on any crypto. Did it with stellar, Bitcoin, Doge, Tezos, and more. Only ones NOT effected are Monero and Cardano because they use their own things

**Comment by gre** [comment]
Ok thanks. It probably means the "internal process" crashes, but it could be for many reasons (usb, internal library,..)

could you Ctrl+E the log on a fresh install and after the error happens? (you can empty the Ledger Live data folder in ~/.config too)

I'll also ask the team tomorrow if anyone have seen the issue.
